packages/coinstac-ui/scripts/build.js

```diff
@@ -50,5 +48,5 @@ rm('coinstac-*')
   });
 })
 .catch((err) => {
-  console.log(`Build failed with: ${err}`); // eslint-disable-line no-console
```

Comment:
Using string literals seems to serialize the error as `err.toString()`. This isn't so useful when debugging build errors.